### PR TITLE
Fix likes initialization in user feed and explore page

### PR DESCRIPTION
**Pull request by javierarce**
This PR fixes the faulty initialization of the likes in both the explore page and the user feed.

### CODE REVIEW

**Comment by javierarce** [comment]
Please have a look, @xavijam

File: lib/assets/javascripts/cartodb/user_feed/view.js [hunk 1]
```diff
@@ -263,23 +251,24 @@ module.exports = cdb.core.View.extend({
...
-      likeModel.url = url;
-      likeModel.sync = Backbone.withCORS;
-      likeModel.fetch();
-    }
+      this._fetchLike(likeModel, url);
+    } else {
+      var self = this;
+      this.options.authenticatedUser.bind('change', function() {
+        self._fetchLike(likeModel, url);
+      });
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/user_feed/view.js]: If you add ```this``` at the end, you won't need the ```self```.

File: lib/assets/test/spec/cartodb/common/views/feed/feed.spec.js [hunk 2]
```diff
@@ -74,10 +74,6 @@ describe('common/views/feed/view', function() {
     expect(this.view.$('.js-more').hasClass('is-hidden')).toBeTruthy();
   });
 
-  it('should init the likes', function() {
```
Comment (xavijam) [inline lib/assets/test/spec/cartodb/common/views/feed/feed.spec.js]: We should add a new test checking if the authenticated user arrives, likes model should have a new url set.

Comment (javierarce) [inline lib/assets/test/spec/cartodb/common/views/feed/feed.spec.js]: Ok

**Comment by xavijam** [comment]
Thanks :+1:

**Pull request merged**